src/exceptions/OCaml-LGPL-linking-exception.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <exception licenseId="OCaml-LGPL-Linking-Exception" name="OCaml LGPL Linking Exception">
```

Comment:
You should probably set [`listVersionAdded`][1] (more docs [here][2]) with a FIXME value or some such, so we don't forget to set that before merging this PR.  The attribute is not required, but I expect that's so we can grandfather in some older entries (where we don't remember when they were added).

[1]: https://github.com/spdx/license-list-XML/blob/v3.1/schema/ListedLicense.xsd#L53
[2]: https://github.com/spdx/license-list-XML/blob/v3.1/schema/ListedLicense.xsd#L135